docs/user/modeling.md

```diff
@@ -167,6 +167,54 @@ Through us defining the model, Nautobot will now be able to dynamically load IP
 !!! note
     Although `Interface.ip_addresses` is a generic relation, there is only one content type (i.e. `ipam.ipaddress`) that may be related through this relation, therefore we don't have to specific this in any way.
 
+## Sorting Relationships
```

Comment:
This looks up-to-date, right? So it should be an accurate representation of how to use this feature?